workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/request/ExecuteRequestExecutor.java

```diff
@@ -45,6 +52,30 @@ public void execute(ActivityExecutorContext<ExecuteRequest> execution) throws IO
     execution.setOutputVariables(outputs);
   }
 
+  private String encodeQueryParameters(String fullUrl) {
+    // splits by the first occurrence of "?" to get the query params part
+    String[] splitUrl = fullUrl.split("\\?", 2);
+    String encodedUrl = splitUrl[0];
+
+    MultiValueMap<String, String> queryParamsMap =
+        UriComponentsBuilder.fromUriString(fullUrl).build().getQueryParams();
+
+    if (!queryParamsMap.isEmpty()) {
+      log.info("Encoding query parameters with Standard UTF-8 format");
```

Comment:
I would put this a debug (and even then, I'm not sure I would log that)

Reply:
You could use https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/util/UriComponentsBuilder.html#replaceQueryParams-org.springframework.util.MultiValueMap-